### PR TITLE
MM-13758 - Updating fa-chevron icon to fa-angle

### PR DESCRIPTION
**Pull request by asaadmahmood**
#### Summary
MM-13758 - Updating fa-chevron icon to fa-angle

#### Ticket Link
https://mattermost.atlassian.net/browse/MM-13758

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [ ] Ran `make check-style` to check for style errors (required for all pull requests)
- [ ] Ran `make test` to ensure unit and component tests passed
- [ ] Added or updated unit tests (required for all new features)
- [ ] Needs to be implemented in mobile (link to PR or User Story)
- [ ] Has server changes (please link)
- [ ] Has redux changes (please link)
- [x] Has UI changes
- [ ] Includes text changes and localization file ([.../i18n/en.json](https://github.com/mattermost/mattermost-webapp/blob/master/i18n/en.json)) updates
- [ ] Touches critical sections of the codebase (auth, posting, etc.)

### CODE REVIEW

File: components/multiselect/multiselect.jsx [hunk 1]
```diff
@@ -264,7 +264,7 @@ export default class MultiSelect extends React.Component {
                 if (this.state.page > 0) {
                     previousButton = (
                         <button
-                            className='btn btn-default filter-control filter-control__prev'
+                            className='btn btn-link filter-control filter-control__prev'
```
Comment (saturninoabril) [inline components/multiselect/multiselect.jsx]: I think this fix is for [MM-13811](https://mattermost.atlassian.net/browse/MM-13811) which has PR submitted - https://github.com/mattermost/mattermost-webapp/pull/2280/files

**Comment by asaadmahmood** [comment]
Done @saturninoabril

File: components/multiselect/__snapshots__/multiselect.test.jsx.snap [hunk 2]
```diff
@@ -299,7 +299,7 @@ exports[`components/multiselect/multiselect should match snapshot for page 2 1`]
     className="filter-controls"
   >
     <button
-      className="btn btn-default filter-control filter-control__prev"
+      className="btn btn-link filter-control filter-control__prev"
```
Comment (sudheerDev) [inline components/multiselect/__snapshots__/multiselect.test.jsx.snap]: This snapshot has to be updated

Comment (asaadmahmood) [reply components/multiselect/__snapshots__/multiselect.test.jsx.snap]: Done.

**Pull request merged**